modal_version/__init__.py

```diff
@@ -1,4 +1,4 @@
 # Copyright Modal Labs 2025
 """Supplies the current version of the modal client library."""
 
-__version__ = "1.0.4.dev4"
+__version__ = "1.0.4.dev5"
```

Comment:
You don't need to change this manually for dev releases; it happens automatically when a PR is merged (https://www.notion.so/modal-com/Client-deployment-workflow-1f51e7f1694980d381d5c5c0f4f1d8fc?source=copy_link)